webapp/package.json

```diff
@@ -11,19 +11,23 @@
   "author": "",
   "license": "",
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
-    "eslint": "^5.2.0",
-    "eslint-import-resolver-webpack": "^0.10.1",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-react": "^7.10.0",
-    "webpack": "^4.16.1",
-    "webpack-cli": "^3.0.8"
+    "@babel/cli": "7.5.0",
```

Comment:
I think there are newer versions of these. Might as well go to latest? 
I usually use npm-check to find the latest versions for me: https://www.npmjs.com/package/npm-check

Reply:
done!